github/pulls_reviews_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestPullRequestsService_ListReviews(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/pulls/1/reviews", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypePullRequestReviewsPreview)
+		fmt.Fprint(w, `[{"id":1},{"id":2}]`)
+	})
+
+	reviews, _, err := client.PullRequests.ListReviews("o", "r", 1)
+	if err != nil {
+		t.Errorf("PullRequests.ListReviews returned error: %v", err)
+	}
+
+	want := []*PullRequestReview{
+		{ID: Int(1)},
+		{ID: Int(2)},
+	}
+	if !reflect.DeepEqual(reviews, want) {
+		t.Errorf("PullRequests.ListReviews returned %+v, want %+v", reviews, want)
+	}
+}
```

Comment:
I found that there are test cases for invalid request for other API.

e.g. https://github.com/google/go-github/blob/4dcd653e8d6b7aa32a6d50d938cecca80a8b4b61/github/pulls_comments_test.go#L71

```go
func TestPullRequestsService_ListComments_invalidOwner(t *testing.T) {
	_, _, err := client.PullRequests.ListComments("%", "r", 1, nil)
	testURLParseError(t, err)
}
```

I think it's not so important test but it might be better to add similar tests for review API as well.